.github/workflows/cargo-test.yml

```diff
@@ -56,7 +56,7 @@ jobs:
           RUST_LOG: info,regalloc=warn
         with:
           command: test
-          args: --release --workspace --features ${{ matrix.metadata }}
+          args: --release --workspace --features parachain-metadata-kintsugi-testnet
```

Comment:
I wanted to run the tests only for `parachain-metadata-kintsugi-testnet` for now, but it looks like instead it causes this exact test to run once for each item in `matrix.metadata`. @ns212 do you know how to fix this?

Reply:
@gregdhill Got this to work now. Let me know if you have any problems with running the tests on only one set of metadatas for now